app_legacy/Helpers/render/achievement.php

```diff
@@ -79,7 +79,11 @@ function renderAchievementTitle(?string $title, bool $tags = true): string
     }
     $title = str_replace('[m]', '', $title);
 
-    return "$title$missableTag";
+    // If we don't strip consecutive spaces, the
+    // browser doesn't collapse them in forum <pre> tags.
+    $title = preg_replace('/\s+/', ' ', $title);
```

Comment:
Does this actually do anything? Won't the whitespace just be a single preceding or trailing space (depending on where the [m] was)? If so, wouldn't `trim()` be easier?

Reply:
With the line removed:
<img width="238" alt="Screenshot 2023-05-05 at 2 20 51 PM" src="https://user-images.githubusercontent.com/3984985/236537714-d5476d10-65a9-48fa-9339-99a3e4703080.png">

With the line changed to `$title = trim($title);`:
<img width="242" alt="Screenshot 2023-05-05 at 2 21 12 PM" src="https://user-images.githubusercontent.com/3984985/236537761-0234533b-c730-4e31-b9db-d67027287dbc.png">

With the line restored:
<img width="226" alt="Screenshot 2023-05-05 at 2 21 23 PM" src="https://user-images.githubusercontent.com/3984985/236537784-7a00a916-01c0-4527-a0a0-1aac5b2a0a82.png">

Reply:
Are the points already part of the title at this point? That's unfortunate.

Reply:
Based on my interpretation of what's happening in `achievementsAvatar()`, yes that appears to be the case.